src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -158,67 +166,54 @@ public static class ClientOperations
+                    int expectedLength = ctx.SetExpectedResponseContentLengthHeader(req.Headers, minLength: 2);
+                    using HttpResponseMessage m = await ctx.SendAsync(req, HttpCompletionOption.ResponseHeadersRead);
 
-                        using (Stream s = await m.Content.ReadAsStreamAsync())
-                        {
-                            s.ReadByte(); // read single byte from response and throw the rest away
-                        }
+                    ValidateHttpVersion(m, ctx.HttpVersion);
+                    ValidateStatusCode(m);
+
+                    using (Stream s = await m.Content.ReadAsStreamAsync())
+                    {
+                        s.ReadByte(); // read single byte from response and throw the rest away
                     }
                 }),
 
                 ("GET Headers",
                 async ctx =>
                 {
-                    using (var req = new HttpRequestMessage(HttpMethod.Get, "/headers"))
-                    {
-                        ctx.PopulateWithRandomHeaders(req.Headers);
+                    using var req = new HttpRequestMessage(HttpMethod.Get, "/headers");
+                    ctx.PopulateWithRandomHeaders(req.Headers);
 
-                        using (HttpResponseMessage res = await ctx.SendAsync(req))
-                        {
-                            ValidateHttpVersion(res, ctx.HttpVersion);
-                            ValidateStatusCode(res);
+                    using HttpResponseMessage res = await ctx.SendAsync(req);
 
-                            // Validate that request headers are being echoed
-                            foreach (KeyValuePair<string, IEnumerable<string>> reqHeader in req.Headers)
-                            {
-                                if (!res.Headers.TryGetValues(reqHeader.Key, out var values))
-                                {
-                                    throw new Exception($"Expected response header name {reqHeader.Key} missing.");
-                                }
-                                else if (!reqHeader.Value.SequenceEqual(values))
-                                {
-                                    string FmtValues(IEnumerable<string> values) => string.Join(", ", values.Select(x => $"\"{x}\""));
-                                    throw new Exception($"Unexpected values for header {reqHeader.Key}. Expected {FmtValues(reqHeader.Value)} but got {FmtValues(values)}");
-                                }
-                            }
-                        }
+                    ValidateHttpVersion(res, ctx.HttpVersion);
+                    ValidateStatusCode(res);
 
+                    // Validate that request headers are being echoed
+                    foreach (KeyValuePair<string, IEnumerable<string>> reqHeader in req.Headers)
+                    {
+                        if (!res.Headers.TryGetValues(reqHeader.Key, out var values))
```

Comment:
Type of `var` isn't obvious here.

```suggestion
                        if (!res.Headers.TryGetValues(reqHeader.Key, out IEnumerable<string> values))
```